src/HPRT/Analytics/views.py

```diff
@@ -66,6 +70,49 @@ class AnalyticResultsView(LoginRequiredMixin, ListView):
     redirect_field_name = 'redirect_to'
     model = Patient
 
+    @staticmethod
+    def get_patient_data(data_list):
+        if len(data_list) == 0:
+            return []
+        scores = []
+        data_list.sort(key=lambda x: x.date)
+        time_delta = datetime.timedelta(weeks=6)
```

Comment:
maybe this time delta number of weeks should be a parameter to the function that defaults to 6?  That way you don't have to change anything else but it's still flexible for updating later